ibc-testkit/src/relayer/context.rs

```diff
@@ -140,12 +143,13 @@ mod tests {
             .host_type(HostType::SyntheticTendermint)
             .latest_height(chain_b_start_height)
             .build()
-            .with_client_parametrized_with_chain_id(
-                chain_id_a,
-                &client_on_b_for_a,
-                client_on_b_for_a_height,
-                Some(mock_client_type()), // The target host chain is mock.
-                Some(client_on_b_for_a_height),
+            .with_client_config(
+                MockClientConfig::builder()
+                    .client_chain_id(chain_id_a)
+                    .client_id(client_on_b_for_a.clone())
+                    .client_state_height(client_on_b_for_a_height)
+                    .client_type(mock_client_type()) // The target host chain is mock.
```

Comment:
What about having the `mock_client_type()` as the default `client_type`?

Reply:
[51f7871](https://github.com/cosmos/ibc-rs/pull/1051/commits/51f78718610c2dd48a2db7ecd32b82506f1dc408)